File: csharp/spaceport/Spaceport/LaunchInfo.cs
using System;

namespace SpacePort
{
    public class LaunchInfo
    {
        public enum LaunchStatus
        {
            Aok,
            Delayed,
            Scrubbed,
            Launched
        };

        public LaunchInfo(Guid guid)
        {
            LaunchID = guid;
        }
        
        private LaunchInfo() { }
        
        public Guid LaunchID { get; set; }
        public string Destination { get; set; }
        public string FlightNumber { get; set; }
        public DateTime Time { get; set; }
        public string Launchpad { get; set; }
        public LaunchStatus Status { get; set; }
    }
}

File: python/spaceport/launch_info.py
from dataclasses import dataclass
import uuid
import datetime
from enum import Enum

class LaunchStatus(Enum):
    AOK = 1
    DELAYED = 2
    SCRUBBED = 3
    LAUNCHED = 4

@dataclass
class LaunchInfo:
    launch_id: uuid
    destination: str
    flight_number: str
    launch_time: datetime
    launchPad: str
    status: LaunchStatus

